src/Analyzers/CSharp/Tests/GenerateDefaultConstructors/GenerateDefaultConstructorsTests.cs

```diff
@@ -1514,7 +1514,8 @@ protected B(int x)
                 {
                 }
             }
-            """);
+            """,
+            index: 1);
```

Comment:
intentional. because code actions were merged, the 0th is now the one in this group that uses the dialog to pick the membes, while the 1nth is the one that creates a delegating constructor.